### PR TITLE
WINC/Health management for Windows nodes RN -- WMCO 7.0.0

### PR DESCRIPTION
**Pull request by mburke5678**
Release Note for https://github.com/openshift/openshift-docs/pull/52788
[
Preview](https://53349--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-7-x.html#wmco-7-0-0)

**Peer Review Squad**:  Review the _New features_, _Bug fixes_, and _Known issues_ sections. The rest is boilerplate. The links to the errata and _Creating a Windows MachineSet object on vSphere_ are not live yet.

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-7-x.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+=== New features and improvements
+
+[id="wmco-7-0-0-wicd"]
+==== Windows Instance Config Daemon (WICD)
+
+The Windows Instance Config Daemon (WICD) is now performing many of the tasks that were previously performed by the Windows Machine Config Bootstrapper (WMCB). The WICD is installed on your Windows nodes. Users do not need to interact with the WICD and should not experience any difference in WMCO operation.
+
+[id="wmco-7-0-0-gcp"]
+==== Support for clusters running on Google Cloud Platform
+You can now run Windows Server 2022 nodes on a cluster installed on Google Cloud Platform (GCP). You can create a Windows `MachineSet` object on GCP to host Windows Server 2022 compute nodes. For more information, see ../windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc#creating-windows-machineset-gcp[Creating a Windows MachineSet object on vSphere].
```
Comment (GroceryBoyJr) [inline windows_containers/windows-containers-release-notes-7-x.adoc:46]: missing an xref keyword. Maybe this is intentional? Your .adoc filename appears in the preview.

File: windows_containers/windows-containers-release-notes-7-x.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+[id="wmco-7-0-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, restarting the WMCO in a cluster with running Windows Nodes caused the windows exporter endpoint to be removed. Because of this, each Windows node could not report any metrics data. After this fix, the endpoint is retained when the WMCO is restarted. As a result, metrics data is reported properly after restarting WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2107261[**BZ#2107261**])
+
+* Previously, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with Windows Defender, regardless of state. This resulted in an error when creating firewall exclusions for containerd on instances without Windows Defender installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed or not. (https://issues.redhat.com/browse/OCPBUGS-3573[**OCPBUGS-3573**])
```
Comment (GroceryBoyJr) [inline windows_containers/windows-containers-release-notes-7-x.adoc:53]: Beware: is OCPBUGS-3573 public or sanitized for public viewing?

**Review by GroceryBoyJr — COMMENTED** [review]
Just a few nits, good work sir.

File: windows_containers/windows-containers-release-notes-7-x.adoc (hunk 2 above)
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-7-x.adoc:53]: I can access the bug without logging in. I assume that means it is public.

File: windows_containers/windows-containers-release-notes-7-x.adoc (hunk 1 above)
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-7-x.adoc:46]: Intentional. We need to hide the WINC files when it releases after the associated OCP version (4.12 in this case).

File: windows_containers/windows-containers-release-notes-7-x.adoc [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
+
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
+endif::openshift-origin[]
+
+[id="wmco-7-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 7.0.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.0.0 were released in https://access.redhat.com/errata/RHSA-2022:9096.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-7-x.adoc:34]: a. the errata is not live yet. 
b. Done!

**Pull request merged**